nuxeo-core/nuxeo-core-management/src/main/java/org/nuxeo/ecm/core/management/statuses/StatusServlet.java

```diff
@@ -28,16 +28,21 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-
 import org.nuxeo.common.Environment;
+import org.nuxeo.ecm.core.management.api.ProbeManager;
 import org.nuxeo.runtime.AbstractRuntimeService;
 import org.nuxeo.runtime.RuntimeService;
 import org.nuxeo.runtime.api.Framework;
 
 /**
  * Servlet for retrieving Nuxeo services running status
+ * 
+ * @since 9.3 
+ * Returns a status based of all the probes registered for the healthCheck 
+ * GET /runningstatus?info=probe&key=probeName returns a healthCheck status based on a probe
```

Comment:
The servlet is not here since 9.3. 
Add a `.` add at then end of the first sentence. 
If we put some documentation in there, then it must be complete and add the other cases (empty GET, POST etc...) We can probably postpone that to document it nicely on doc.nuxeo.com